src/main/java/com/backbase/ct/bbfuel/configurator/BillPayConfigurator.java

```diff
@@ -32,4 +40,36 @@ public void ingestBillPayUser(String externalUserId) {
             log.info("Bill Pay user [{}] ingested", externalUserId);
         }
     }
+
+    public void ingestBillPayAccounts(String subscriberId, List<Account> accounts) {
+        loginRestClient.login(subscriberId, subscriberId);
+        userContextPresentationRestClient.selectContextBasedOnMasterServiceAgreement();
+        UserByIdPutRequestBody request = new UserByIdPutRequestBody()
+            .withSubscriber(createSubscriber(subscriberId, accounts));
+        Response response = billPayMockBankRestClient.ingestAccounts(request);
+        if (isBadRequestException(response, "The request is invalid")) {
+            log.info("Bad request for Bill Pay user [{}]", subscriberId);
+        } else {
+            response.then().assertThat().statusCode(SC_OK);
+            log.info("Accounts ingested for Bill Pay user [{}]", subscriberId);
+        }
+    }
+
+    private Subscriber createSubscriber(String subscriberId, List<Account> accounts) {
+        return new Subscriber()
```

Comment:
why not generate differentiated values?

Reply:
Any non-account data isn't used or stored anywhere when using the mock BillPay provider so theres no need to randomise the data